### PR TITLE
Event page

### PR DESCRIPTION
**Pull request by MarceColl**
- Added static page for quick edition of event/talks pages
  - Use Control+E at static.html to access edition, again to close it
- Updated background animation to use WebGL
- Updated /live with:
  - Live reloading
  - Progress bars
  - Top links
  - Desktop notifications
  - Options to unsubscribe to some types of notifications
  - Option to stop background animation

To try, download a fresh copy of this branch and execute

``` shell
npm install
bower install
gulp
```

live is at `/live.html`
static is at `/static.html`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,14 +11,33 @@ globally. Then run:
...
+New features:
+
+- Desktop notifications
+- Live reload
+- Options panel to filter by type of notifications
+  - 4 types of notifications (provisional): essential, food, events, talks
+  - essential notifications are always sent
+  - food, events and talks can be filtered out depending on your preferences.
+- Options panel also includes a checkbox to switch off background animation.
+- Progress bars!
```
Comment (casassg) [inline README.md:43]: I was talking about a progress bar of the full event (or a countdown clock of how many hours missing until the end of Hacking). Looks good though.

File: dist/index.html [hunk 2]
```diff
@@ -28,8 +27,7 @@
         <div class="row top-buttons">
             <div class="col-md-12">
                 <div class="button-container">
-                    <a href="https://hackersatupc.typeform.com/to/Bt9YQF">Apply</a>
-                    <a href="mailto:sponsor@hackupc.com">Sponsor</a>
+                    <h2>Applications are closed. See you all at HackUPC</h2>
```
Comment (casassg) [inline dist/index.html]: THis will go in PR #35 as it needs to be finished today. I will use it from here.

File: dist/live.html [hunk 3]
```diff
@@ -5,119 +5,104 @@
...
   <body>
-    <h1># HACKUPC Live ヽ(◕‿◕)ﾉ <span id="title-date"></span></h1>
-    <div id="timetable"></div>
-
-    <div id="twitter-container">
-        <a class="twitter-timeline" height="100%" href="https://twitter.com/search?q=%28hackupc%20OR%20from%3Ahackupc%29%20AND%20-filter%3Aretweets" data-widget-id="700696875549102080" data-link-color="#CE481C">Tweets about (hackupc OR from:hackupc) AND -filter:retweets</a>
-        <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+    <div></div>
+    <div id="app">
+        <div class="navbar col-sm-12" id="livenav">
```
Comment (casassg) [inline dist/live.html]: position absolute -> fixed to the top.

File: dist/live.html [hunk 4]
```diff
@@ -5,119 +5,104 @@
...
+                        <li class="active col-sm-2"><a href="#">Maps</a></li>
+                        <li class="active col-sm-2"><a href="#">LIVE</a></li>
+                        <li class="active col-sm-2"><a href="#">The Game</a></li>
+                        <li class="active col-sm-2"><a href="#">MLH Hardware Lab</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <h1># HACKUPC Live ヽ(◕‿◕)ﾉ <span id="title-date"></span> <span v-on:click="options" class="fa fa-cog"></span></h1>
```
Comment (casassg) [inline dist/live.html]: Add small countdown of remaining hacking hours 
- Before the event -> 36h
- During the Event -> countdown
- After the event -> Link to submit you hack!

File: dist/live.html [hunk 5]
```diff
@@ -5,119 +5,104 @@
...
+        <div class="navbar col-sm-12" id="livenav">
+            <div class="container-fluid">
+                <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                    <ul class="nav navbar-nav col-md-12">
+                        <li class="active col-sm-2 col-sm-offset-1"><a href="#">Welcome</a></li>
+                        <li class="active col-sm-2"><a href="#">Maps</a></li>
+                        <li class="active col-sm-2"><a href="#">LIVE</a></li>
+                        <li class="active col-sm-2"><a href="#">The Game</a></li>
+                        <li class="active col-sm-2"><a href="#">MLH Hardware Lab</a></li>
+                    </ul>
```
Comment (casassg) [inline dist/live.html:35]: To be added -> Devpost (hackupc2016f.devpost.com)

File: dist/live.html [hunk 6]
```diff
@@ -5,119 +5,104 @@
...
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label>
+                            <input type="checkbox" v-model="eventsNotify"> Events
+                        </label>
+                    </div>
+                    <div class="checkbox" style="display: none;">
+                        <label>
+                            <input type="checkbox" v-model="bieneNotify"> Biene
```
Comment (casassg) [inline dist/live.html:98]: CUANDO BIENE?

File: dist/static.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  <meta name="author" content="Hackers@UPC">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+  <link rel="stylesheet" href="assets/css/static.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Ek+Mukta" rel="stylesheet">
+</head>
+
+<body>
+    <div id="app">
+      <h1 class="col-sm-4 col-sm-offset-4" id="title">{{title}}</h1>
```
Comment (casassg) [inline dist/static.html:21]: It's too low and too small. Needs to be centered as HackUPC banner in index.html

File: dist/static.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+          (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-72515524-1', 'auto');
+      ga('send', 'pageview');
+
+    </script>
+    <span style="display: none;">BIENE HACKUPC!</span>
```
Comment (casassg) [inline dist/static.html:47]: More cowbell!

File: schedule.py [hunk 9]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python
```
Comment (casassg) [inline schedule.py]: Add documentation about how to use this pls

File: src/scripts/leaves.ts [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+    if(this.paused == false) {
+      this.drawLeaves();
+      this.update(dt);
+    }
+  }
+
+  private loadLeaves() {
+    PIXI.loader
+      .add("leaf0", "assets/img/leaf0.png")
```
Comment (casassg) [inline src/scripts/leaves.ts:118]: for some reason, it only loads 2 types of leaves

**Review by casassg — CHANGES_REQUESTED** [review]

File: dist/index.html (hunk 2 above)
Comment (casassg) [reply dist/index.html]: Add button to Live

File: src/scripts/live.js [hunk 11]
```diff
@@ -1,119 +1,177 @@
...
+var leaves;
 
-var template
-var lastHalf
-var titleDate = document.getElementById('title-date')
-var timetable = document.getElementById('timetable')
+// Testing flag, for mocking the dates of the events
+// without having to change the dates in the events.json
+var testing = true;
+var first = true;
```
Comment (casassg) [inline src/scripts/live.js]: What's this for?

Comment (MarceColl) [reply src/scripts/live.js]: the testing variable skips the dates of the events.json and creates the events in the near future for testing purposes

File: src/scripts/leaves.ts (hunk 10 above)
Comment (MarceColl) [reply src/scripts/leaves.ts:118]: Fixed in  eee6cf9

File: schedule.py (hunk 9 above)
Comment (MarceColl) [reply schedule.py]: Will do, it wasnt even supposed to be uploaded yet hahaha, it's a WIP

**Comment by MarceColl** [comment]
Updated with new changes

**Review by casassg — COMMENTED** [review]
LGTM now

**Review by casassg — APPROVED** [review]

**Pull request merged**